src/components/dashboard/styles/index.css

```diff
@@ -381,3 +381,12 @@ nav {
 #cardCapitalize{
   text-transform: capitalize;
 }
+.dashboard-alert{
+  background: #e9e9e9;
+  color: #000000!important;
+  text-align: center;
+}
+.search-filter {
+  padding-right: 5%;
+  float: right;
+}
```

Comment:
Could you add a line at the end of the file

Reply:
resolved